### PR TITLE
WINC-1208: Remove unsupported versions

### PR DESCRIPTION
**Pull request by saifshaikh48**
This PR removes all unsupported versions of WMCO and WMCB to date.

The latest is WMCO v6, which went [EOL](https://access.redhat.com/support/policy/updates/openshift_operators) on Feb 10, 2024 with [OCP 4.11](https://access.redhat.com/support/policy/updates/openshift).
Older WMCO releases, those mapped to OCP versions 4.4, 4.5, 4.6, 4.7, 4.8, 4.9 and 4.10, are also removed.

WMCB is removed entirely as WMCO v6 was the last release it was shipped in.

### CODE REVIEW

File: ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.11__okd.yaml [hunk 1]
```diff
@@ -1,75 +0,0 @@
-base_images:
```
Comment (sebsoto) [inline ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.11__okd.yaml:1]: Its a bit confusing that this is being done for 4.11 when there are 4.10, etc being left.
Can we remove the entire WMCB repo and WMCO 4.11,4.10, etc?

If so please reframe this as `Remove unsupported versions`

**Review by sebsoto — CHANGES_REQUESTED** [review]

**Comment by saifshaikh48** [comment]
had to push an update to resolve a merge conflict that arose overnight

**Comment by saifshaikh48** [comment]
@alinaryan thank you for helping get this in before another merge conflict arose :rocket:

**Pull request merged**